javascript.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    extends: [ 'airbnb-base' ],
+    rules: {
+        'import/prefer-default-export': 'off'
+    }
+};
```

Comment:
No newline. Maybe we should lint sources in this repo too? :)

Reply:
Are you serious? :)

Reply:
Why not?